math/mathcore/inc/TMath.h

```diff
@@ -259,36 +266,40 @@ constexpr Double_t Kcgs()
 /// Boltzmann's constant uncertainty.
 constexpr Double_t KUncertainty()
 {
-   return 0.0000024e-23;
+   // constant is exact according to 2019 redefinition 
+   // (https://en.wikipedia.org/wiki/2019_redefinition_of_the_SI_base_units)
+   return 0.0;
 }
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Stefan-Boltzmann constant in \f$ W m^{-2} K^{-4}\f$
 /// \f[ \sigma \f]
 constexpr Double_t Sigma()
 {
-   return 5.6704e-8;
+   return 5.670373e-8;
 }
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Stefan-Boltzmann constant uncertainty.
 constexpr Double_t SigmaUncertainty()
 {
-   return 0.000040e-8;
+   return 0.0;
```

Comment:
What do we do if somebody divides by this constant? This will break their code. Maybe this has to be mentioned in the release notes?

Reply:
Yes, we should mention it there. Hopefully one multiplies and not divides by the uncertainty